app.go

```diff
@@ -155,6 +158,59 @@ func getApp(client *cmd.Client, appName string) (app, error) {
 	return a, err
 }
 
+func setEnvs(client *cmd.Client, appName string, envVars *api.Envs) error {
+	url, err := cmd.GetURL("/apps/" + appName + "/env")
+	if err != nil {
+		return err
+	}
+	v, err := form.EncodeToValues(envVars)
+	if err != nil {
+		return err
+	}
+	req, err := http.NewRequest("POST", url, strings.NewReader(v.Encode()))
+	if err != nil {
+		return err
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+	resp, err := client.Do(req)
+	if err != nil {
+		return err
+	}
+	return cmd.StreamJSONResponse(ioutil.Discard, resp)
+}
+
+func getEnvs(client *cmd.Client, appName string) ([]envVar, error) {
+	var vars []envVar
+	resp, err := doReq(client, "/apps/"+appName+"/env")
```

Comment:
There's an explanation: https://groups.google.com/forum/#!msg/golang-nuts/Uqu6IVLDSeI/Fqv6QLH-zkgJ.

I don't buy it, but I don't think this is going to change x)